postcss.config.cjs

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable global-require */
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
why are you disabling the second eslint rule?

Reply:
thanks for asking, the purpose of this was to disable require over eslint but i found the i can use only one line